packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -116,6 +116,10 @@ export class SerializedStateManager {
 					eventName: "getSnapshotTreeFailed",
 					id: version.id,
 				});
+			} else if (snapshot !== undefined && version === undefined) {
+				this.mc.logger.sendErrorEvent({
```

Comment:
i like error, as this is only fatal on the summarizer client. the regular client should be fine